### DISCUSSION TITLE
problems installing X11iraf

### DISCUSSION THREAD

**Discussion by mgregg128** [iraf-community/iraf#298] [Q&A] [answered]
Hello,
I am having trouble installing x11iraf on my MacBook pro (intel, 2017) running Ventura (13.3).  I have the latest Xcode installed.  When I download the x11iraf tarball and attempt to run make, I get this message:
```
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C obm
cc -I/usr/include/tcl -I/usr/X11/include   -c -o Obm.o Obm.c
In file included from Obm.c:5:
./ObmP.h:45:10: fatal error: 'X11/Xaw3d/Layout.h' file not found
#include <X11/Xaw3d/Layout.h>
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.
make[1]: *** [Obm.o] Error 1
make: *** [obm/libobm.a] Error 2
```
And, indeed, there is no such file (or directory) /usr/X11/Xaw3d on my system.  /usr/X11 is a softlink:
/usr/X11@ -> ../private/var/select/X11

The missing Xaw3D directory and Layout.h file on my system are here:
/opt/X11/include/X11/Xaw3d/etc.

Before I go making softlinks all over to try to chase this and possible other missing files, I am hoping to get advice or instructions from those who can diagnose this.  
Thank you for the help,
Michael

**Comment by olebole** [comment, marked as answer]
Did you install the "XQuartz" package as described in the `README.md`?

**Comment by mgregg128** [comment]
Hello,

That did it!  I had ignored installing xquartz because I already had it running and it was functioning normally with xterm.  Updating to 2.8.5 was necessary for xgterm.  Apologies for skipping that step!

Thank you very much for the help.  

One last question:  Perhaps this is wishing for the Moon, but is there any possibility to get STSDAS or SYNPHOT installed and running within iraf?  What is their status?  It would be very useful to have the drizzlepac routines available again in iraf/pyraf.  

Thank you again!

Michael

**Reply by olebole** [reply]
STSDAS is 32-bit only, so it won't work on modern Macs, unless someone ports it. It also has major licensing issues (code from Numerical Recipes), which would need to be resolved as well.
